packages/components/badge/src/index.ts

```diff
@@ -1,5 +1,5 @@
-export { Badge } from './Badge';
-export type { BadgeProps } from './Badge';
-export { EntityStatusBadge } from './EntityStatusBadge';
-export type { EntityStatusBadgeProps } from './EntityStatusBadge';
+export { Badge } from './Badge/Badge';
+export type { BadgeProps } from './Badge/Badge';
```

Comment:
I thought these two would stay in the src folder, no?

Reply:
but we have 2 separate README files for them. how do you think we should handle those cases?

Reply:
Ah I sent this messages before we talked this morning

they are not relevant anymore :)